mixer/pkg/server/args.go

```diff
@@ -121,6 +125,7 @@ func NewArgs() *Args {
 		ConfigIdentityAttribute:       "destination.service",
 		ConfigIdentityAttributeDomain: "svc.cluster.local",
 		ConfigFetchIntervalSec:        5,
+		LoggingOptions:                log.NewOptions(),
```

Comment:
Do we need this to be "String()"ed and/or validated as well?

Reply:
Added to the String method. Validation is performed when the options are used.